calico_containers/calico_ctl/node.py

```diff
@@ -109,8 +109,12 @@ def validate_arguments(arguments):
     asnum_ok = True
     if arguments.get("<AS_NUM>") or arguments.get("--as"):
         try:
-            asnum = int(arguments["<AS_NUM>"] or arguments["--as"])
-            asnum_ok = 0 <= asnum <= 4294967295
+            asnum = str(arguments.get("<AS_NUM>") or arguments.get("--as"))
```

Comment:
Maybe pull this outside so that you don't have to repeat the gets in the if statement immediately above.